Examples/Physics_applications/pierce_diode/README.rst

```diff
@@ -0,0 +1,60 @@
+.. _examples-pierce-diode:
+
+Pierce Diode at the Child–Langmuir Limit
+===================
+
+This example shows how to simulate the physics of a 1D Pierce diode configuration operating
+at the Child–Langmuir limit using WarpX. In this setup, an electron beam is injected
+into a planar diode gap, consisting of by parallel conducting plates separated by the distance :math:`d` and powered by a voltage difference :math:`V`.
+For a suitable voltage, the field accelerates charged particles initially at rest (:math:`v = 0`) at the cathode towards the anode.
```

Comment:
I am not sure I understand "For a suitable voltage" here. The acceleration of charged particles towards the anode would happen for any positive voltage here, no?